devops/scripts/benchmarks/compare.py

```diff
@@ -46,6 +46,24 @@ class BenchmarkHistoricAverage:
     # TODO Ensure ONEAPI_DEVICE_SELECTOR? GPU name itself?
 
 
+class OutputFile:
```

Comment:
Also, just thinking out loud, do we really need this class? As an alternative, we can have a string to keep Github summary and at the end, we can just append this string to the `github_summary.md` file?